script/setup.js

```diff
@@ -36,42 +35,54 @@ try {
 		strict: false,
 	});
 
-	async function getPrefillOrPromptedValue(key, message) {
+	async function getPrefillOrPromptedValue(key, message, placeholder) {
 		const { [key]: value } = values[key]
 			? (console.log(chalk.grey(`Pre-filling ${key} to ${values[key]}.`)),
 			  values)
-			: await prompt({
+			: await text({
 					message,
-					name: key,
-					type: "input",
+					placeholder,
+					validate: (val) => {
+						if (val.length === 0) return "Please enter a value.";
+					},
 			  });
 
+		if (isCancel(value)) {
+			cancel("Operation cancelled.");
```

Comment:
This text is informative but kind of boring. How about we make it a little more friendly? Maybe...

```suggestion
			cancel("Operation cancelled. Existing setup - maybe another time? 👋");
```